docs/release_notes/ibexa_dxp_v4.3.md

```diff
@@ -122,3 +122,30 @@ The `TaxonomyEntryId` Search Criterion is not available in Legacy search Engine.
 | Ibexa Content  | Ibexa Experience  | Ibexa Commerce |
 |--------------|------------|------------|
 | [Ibexa Content v4.3](https://github.com/ibexa/content/releases/tag/v4.3.0) | [Ibexa Experience v4.3](https://github.com/ibexa/experience/releases/tag/v4.3.0) | [Ibexa Commerce v4.3](https://github.com/ibexa/commerce/releases/tag/v4.3.0)|
+
+## v4.3.1
+
+### New REST API endpoints 
+
+You can now use new routes to REST API that confirm if the User is logged in,
+without invoking any other route:
+
+- GET `/user/current` - redirects to current User load API.
+- GET `/user/sessions/current` - returns a current User Session object.
+
+You can retrieve, add and remove users from a segment with:
+
+- GET `/user/users/:userId/segments` - retrieves Segments for a given User.
+- POST `/user/users/:userId/segments` - assigns User to one or more Segments.
+- DELETE `/user/users/:userId/segments/:segmentIdentifier` - unassigns User from a Segment.
+
+You can retrieve the defined languages with:
+
+- GET `/languages`- returns a defined language list.
+- GET `/languages/{languageCode}` - returns a single language.
+
+### New service for token-based authentication
+
+The new release adds `Ibexa\Contracts\Rest\Security\AuthorizationHeaderRESTRequestMatcher` service that can be used instead of `Ibexa\AdminUi\REST\Security\NonAdminRESTRequestMatcher` service.
```

Comment:
```suggestion
The new release adds `Ibexa\Contracts\Rest\Security\AuthorizationHeaderRESTRequestMatcher` service that can be used instead of `Ibexa\AdminUi\REST\Security\NonAdminRESTRequestMatcher`.
```